docs/documentation/release_notes/topics/26_3_0.adoc

```diff
@@ -26,6 +26,10 @@ In this release, the Instagram Identity Broker is deprecated for removal and is
 If you are using this broker, it is recommended to use the Facebook Identity Broker instead. For more
 details, see link:{upgradingguide_link}[{upgradingguide_name}].
 
+= Inconsistent user endpoints harmonized
+
+In previous releases there was an inconsistency in the REST endpoint result of getting a user and searching for a user. When BruteForce is enabled and a user was temporarily locked out one endpoint would return enabled=false while the other would return enabled=true. If the user was updated and enabled was false due to temporary lockout then the user would be disabled permanently. Both endpoints now return enabled=true when a user is temporarily locked out. To check whether a user is temporarily locked out the BruteForeUserResource endpoint should be utilised.
```

Comment:
I would remove this and just set the comment in the upgrading notes. If not we are adding all upgrading notes to the release notes. But this is just my opinion, so no strong opinion here.